plugins/module_utils/lxd.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, msg, **kwargs):
 
 
 class LXDClient(object):
-    def __init__(self, url, key_file=None, cert_file=None, debug=False):
+    def __init__(self, url, key_file=None, cert_file=None, server_cert_file=None, debug=False):
```

Comment:
Better add this as the last argument, to avoid breaking existing code which assumes that the fourth parameter is `debug`:
```suggestion
    def __init__(self, url, key_file=None, cert_file=None, debug=False, server_cert_file=None):
```

Reply:
Indeed, thanks!